aula-pratica-02/ex1.c

```diff
@@ -0,0 +1,23 @@
+#include<stdio.h>
+#include<stdlib.h>
+int primo(int n);
+
+int primo(int n)
+{
+    for (int i = 2; i < n; i ++) {
+        if (n % i == 0) {
+            printf("%d n�o � n�mero primo", n);
+            return 0;
+        }
+    }
+    printf("%d � um n�mero primo", n);
+    return 1;
+}
+
+int main()
+{
+    int n;
+    scanf("%d", &n);
+    primo(n);
+    return 1;

```

Comment:
with the main function, you genereally want to always return 0, because any number returned from `main` that is different than `0` is considered to be an error, so when you return `1`, the OS assumes (and tell the user) that the program ended with an error.

```suggestion
    return 0;
```